Revise code block guidelines, add wkls (#89)
wherobots/wherobots-examples: CONTRIBUTING.md

### Prose/Markdown style

#### Technical points

- Opening image
  - If this is a solution notebook, start with a banner that repeats the imagery used for the notebook's thumbnail. It should be about 1200x220 px. (See [Generate PMTiles using Wherobots](https://github.com/wherobots/wherobots-examples/blob/main/Analyzing_Data/PMTiles-railroad.ipynb) for an example.)
  - Otherwise, start with this small logo: `![Wherobots logo](../assets/img/header-logo.png)`
- Headings
  - Use `#` (H1) once in a notebook, as the title that immediately follows the opening image.
  - Use `##` (H2) for most section headings, and `###` (H3) if a second level makes sense.
  - Anything smaller should probably be omitted or simply emphasized with bold text.
  - If there's text that comes after the heading, put them in the same cell.

> - Put code blocks in markdown cells *only* when it is not also used in one of the code cells.

Reviewer: What's the rationale behind this?

Does this apply to code that includes placeholders and then reinforced by actual executable code?

Reviewer 2: I think the point is succinctness and just refers to not putting the same code cells in md and code cells.

we sometimes want to show the code tha tis long running without executing. 

@copilot make this section clearer

  - Examples: a Pythonic alternative to SQL or something that should be avoided

>   - Short excerpts like function or variable names are fine, and should get `code formatting`.

Reviewer: ```suggestion
  - Function or variable names should be enclosed in `code formatting`.
```

- Use bold only for
  - Presenting or defining a new term.
  - Creating hierarchy when adding another heading level would get cluttered.
- Use `>` indents for definitions.

#### Writing style

- Focus on the scenario at hand. Use links to docs or other resources for expanded coverage of a topic.
  - When linking to docs, remove any `?h=foo` query string parameters. (This turns off any search term highlighting.)
- Focus on education and reader empowerment. Avoid a promotional tone.
- Use few or no emojis.
  - Never use emojis instead of markdown-formattted bullets or numbers. Emoji bullets can impede accessibility for people using screen readers.

### Code blocks and style

> - Precede most code cells with with a markdown cell with a `##` H2 or `###` H3 headline to facilitate rapid scanning.

Reviewer: ```suggestion
- Precede code cells with with a markdown cell with a `##` H2 or `###` H3 headline to facilitate rapid scanning.

For example:

> ### Isochrone implementation example
> 
> ```
> ST_Isochrone(geometry, 1, 'car', false)
> ```

```
